NEWS

```diff
@@ -2,6 +2,15 @@
 
 Sandia OpenSHMEM NEWS -- history of user-visible changes.
 
+v1.4.0rc2
+---------
+- Added support to OFI transport for sharing STX resources across contexts.
+- Added several environment variables that can be used to control STX resource
+  management in the OFI transport: SHMEM_OFI_STX_MAX, SHMEM_OFI_STX_ALLOCATOR,
+  and SHMEM_OFI_STX_THRESHOLD.  See README for details.
+- Added support for returning gracefully when context creation is unsuccessful.
+- Updated data segment exposure method to improve compatibility with tools.
+
```

Comment:
Do we want to include the memory ordering/barrier improvements?

Reply:
Those are already noted in the RC1 changes.  Is there a new change in RC2 that I'm forgetting about?

Reply:
Oh, whoops.  Nope, looks good!